dbus_service.py

```diff
@@ -382,18 +379,14 @@ def store_for_later_use(self, meter_data):
     def check_and_enrich_ahoy_data(self, meter_data):
         ''' Check if Ahoy data is valid and enrich it with additional data'''
         if not "iv" in meter_data:
-            logging.info("You do not have the latest Ahoy Version to run this script,"
-                "please upgrade your Ahoy to at least version 0.5.93")
```

Comment:
Are errors logged automatically?  If so, good work!

Reply:
Yes, in the `_update` function, there is a `except ValueError` which logs it. And I verified it in a short experiment.